ansible_runner/config/_base.py

```diff
@@ -193,6 +193,17 @@ def _prepare_env(self, runner_mode='pexpect'):
             self.env = {}
             # Special flags to convey info to entrypoint or process in container
             self.env['LAUNCHED_BY_RUNNER'] = '1'
+
+            # A bug in the fuse-overlayfs driver in RHEL < 5.8 causes errors when trying to set extended file attributes.
```

Comment:
5.8?!

Reply:
Is there a way to make this conditional on system capabilities? Rhel version? Rather than make it a blanket change on any containerized execution?

Reply:
Doh! Too many numbers. It's the 5.x kernel and the RHEL 8.x version. 🤦‍♂️

Reply:
That was my first inclination too: probe the running image to determine if the setting is needed.

Reply:
I'm not sure if we would be able to get details on the filesystem driver or container host data from within the running container, though. I'm open to suggestions. My endeavors with fact collection inside containers results in mountains of conditional hacks and workarounds.

Reply:
This code in particular, though, exists outside of the container as it's in the environment preparation.

Reply:
Oh, ok. There is hope, then.

Reply:
This may be complicated to discover, as there are several layers where users could have configured their default storage driver.  Setting it regardless for podman shouldn't be too problematic, it just allows 1 additional fallback to perform an unsafe write when `atomic_move` fails.  Currently `atomic_move` has a waterfall of 4 methods, and if one succeeds, even with unsafe writes enabled, it'll never make it to the unsafe write.

For ref, RHEL8.5 is supposed to include the backported kernel fix that is going 5.13 of the kernel.  RHEL8 is using a 4.18 kernel, so that detail is probably not worth mentioning unless we find the exact kernel version they will include the backport in.

https://www.redhat.com/sysadmin/podman-rootless-overlay

But as @shanemcd mentions in his comment, we should probably just make it conditional to running under podman.

Reply:
I will work on setting this only for `podman`.